### PR TITLE
fix(stock): refresh inventory adjustment on submit

### LINKED ISSUE #5255: Automatically refresh the page after a stock adjustment has been made.
Our users are getting frustrated with the stock adjustment module as they don't have a way to refresh the data after a stock adjustment has been made.  Right now, they need to exit the module, and re-enter the module to get the data to refresh.

### PR DESCRIPTION
**Pull request by jniles**
Refreshes the inventory adjustments page on submit so that users don't have to click the refresh button again to get current values in stock.

Closes #5255.

### CODE REVIEW

File: client/src/modules/stock/inventory-adjustment/inventory-adjustment.js [hunk 1]
```diff
@@ -129,8 +129,9 @@ function StockInventoryAdjustmentController(
   function startup() {
     vm.movement = {
       date : new Date(),
-      entity : {},
```
Comment (jmcameron) [inline client/src/modules/stock/inventory-adjustment/inventory-adjustment.js:132]: Why was this removed?

**Review by jmcameron — APPROVED** [review]
Code changes look fine.
I tested this with bhima_test and it works as advertised.

File: client/src/modules/stock/inventory-adjustment/inventory-adjustment.js (hunk 1 above)
Comment (jniles) [reply client/src/modules/stock/inventory-adjustment/inventory-adjustment.js:132]: Good question.  It was never used or needed.  I think it is copy/paste from somewhere where it was needed.

**Review by mbayopanda — APPROVED** [review]
LGTM 👍 

bors r+

**Pull request merged**